### ISSUE TITLE
Something is not working on pool creation

### ISSUE THREAD

**Issue by codenamejason** [allo-protocol/allo-v2-spec#52]
This morning around 9:30 EST when going over the workshop for the Allo SDK I ran into an error when demonstrating creating a pool on Allo. We had previously created a profile with success. For added context, I have been creating pools all week (12/10 - 12/14) and had no issues.

Now when trying to create a pool we get an error:

![Image](https://github.com/allo-protocol/allo-v2-sdk/assets/9419140/b5264f60-85ad-4d46-8ff3-54ecdc07cb75)

This error is vague at best and we will have to investigate this further.

tl;dr This is not a gas error.

**Comment by thelostone-mc** [comment]
This works when using a custom RPC. Solution is to shift to sepolia 

- https://github.com/allo-protocol/allo-v2-spec/pull/51
- https://github.com/allo-protocol/SeaGrants/pull/98

**Comment by 0xKurt** [comment]
This is blocked by spec, right? @thelostone-mc
